releases/release-1.10/README.md

```diff
@@ -0,0 +1,51 @@
+# Kubeflow 1.10
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://www.kubeflow.org/docs/about/community/#kubeflow-community-calendars)
```

Comment:
This link, should be fixed with: https://www.kubeflow.org/docs/about/community/#kubeflow-community-meetings